tests/sanity/test_velero.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright 2024 Canonical, Ltd.
+#
+
+import os
+
+import utils
+
+
+def test_velero_rock():
+    """Test Velero rock."""
+
+    image_variable = "ROCK_VELERO"
```

Comment:
Are you going to test multiple versions? If so, you may want to use this:

https://github.com/canonical/multus-rock/blob/13d369539b4528825182f69fdaab829f9d267ecb/.github/workflows/pull_request.yaml#L21C3-L22C92

then the env variable will be named ``ROCK_VELERO_${version}``

Reply:
Done